client/app/bundles/course/assessment/submission/components/ScribingView/fields/ColorPickerField.jsx

```diff
@@ -66,33 +70,60 @@ const ColorPickerField = (props) => {
   const {
     intl, colorPickerColor, onClickColorPicker,
     colorPickerPopoverOpen, colorPickerPopoverAnchorEl,
-    onRequestCloseColorPickerPopover,
-    onChangeCompleteColorPicker,
+    onRequestCloseColorPickerPopover, onChangeCompleteColorPicker,
+    noFillValue, noFillOnCheck, noFillLabel,
   } = props;
 
+  const rgbaValues = colorPickerColor.match(/^rgba\((\d+),(\d+),(\d+),(.*)\)$/);
+
   return (
-    <div style={styles.colorPickerFieldDiv}>
-      <label htmlFor="color-picker" style={styles.label}>{intl.formatMessage(translations.colour)}</label>
-      <div
-        role="button"
-        tabIndex="0"
-        style={{ background: colorPickerColor, ...styles.colorPicker }}
-        onClick={onClickColorPicker}
-      />
-      <Popover
-        style={styles.toolDropdowns}
-        open={colorPickerPopoverOpen}
-        anchorEl={colorPickerPopoverAnchorEl}
-        anchorOrigin={popoverStyles.anchorOrigin}
-        targetOrigin={popoverStyles.targetOrigin}
-        onRequestClose={onRequestCloseColorPickerPopover}
-        animation={PopoverAnimationVertical}
-      >
-        <SketchPicker
-          color={colorPickerColor}
-          onChangeComplete={onChangeCompleteColorPicker}
+    <div>
+      <div>
+        { noFillOnCheck ?
+          <Checkbox
+            label={noFillLabel}
```

Comment:
Are you intending to make this a generic checkbox? If it's only going to be for "no fill", maybe directly use the translation in this component instead of passing the translated string all the way down? `scribingTranslations` is already being imported, we get can get that for free

Reply:
Yep, it's only used for no fill. I'll do that